the-grid.html

```diff
@@ -149,6 +149,17 @@
                         value: false,
                         reflectToAttribute: true,
                         observer: '_attachEvents'
+                    },
+
+                    /**
+                     * Enable infinite automatic extension of the grid on resize and move.
+                     * @type {boolean}
+                     */
+                    selfResize: {
```

Comment:
@vpusher well how would that work though. I'm updating that field, so right after the first update of the field, it wouldn't be dynamic anymore. Or am I missing something?